EasyRestProjectNetTeam2/EasyRestTests/CheckNavigationTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿using EasyRestProjectNetTeam2.EasyRestPages;
+using NUnit.Framework;
+
+namespace EasyRestProjectNetTeam2.EasyRestTests
+{
+    class CheckNavigationTests : BaseTest
+    {
+        HomePage homePage;
+        SignInPage signInPage;
+        RestaurantsPage restaurantsPage;
+        MenuPage menuPage;
+
+        [Test]
+        public void CheckNavigationToMenuCategoryHot()
+        {
+            homePage = GetHomePage();
+            homePage.HeaderMenuComponent.ClickSignInButton();
+            signInPage = GetSignInPage();
+            signInPage.SendKeysToInputEmail(dataModel.Email);
+            signInPage.SendKeysToInputPassword(dataModel.Password);
+            homePage.HeaderMenuComponent.ClickSignInButton();
```

Comment:
You have duplicated logic - sign-in steps are present for all three tests. Please move this logic to the setup method.

Reply:
Thnx. I've added one method for this.

Reply:
How did you resolve this? I don't see it.

Reply:
![Screenshot_14](https://user-images.githubusercontent.com/103622690/182572786-42cf1e52-b2bd-4b2f-8e01-beee46311d05.png)